app/controllers/api/base_controller.rb

```diff
@@ -281,8 +282,10 @@ def assign_lone_taxonomies
     end
 
     def add_version_header
-      response.headers["Foreman_version"] = SETTINGS[:version].full
-      response.headers["Foreman_api_version"] = api_version
+      response.headers['Foreman_api_version'] = SETTINGS[:api_version]
```

Comment:
We shouldn't modify `api_version` -- that's just "2" and clients (even before authentication) could use that info.